### PR TITLE
[INLONG-5551][DataProxy] Add metric log output information in TubeSink

### LINKED ISSUE #5551: [Improve][DataProxy] Add metric log output information in TubeSink
### Description

Add metric log output information in TubeSink

TubeSink currently does not provide the indicator log output function, after startup, the information sent by TubeSink cannot be queried from the index.log and monitor.log log files. This issue is to make this improvement and add metric log output information in TubeSink

### InLong Component

InLong DataProxy

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gosonzhang**
- Fixes #5551

### CODE REVIEW

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java [hunk 1]
```diff
@@ -121,6 +133,10 @@ public void configure(Context context) {
         // create producer holder
         producerHolder = new TubeProducerHolder(getName(),
                 usedMasterAddr, configManager.getMqClusterConfig());
+        // get statistic configure items
+        maxMonitorCnt = context.getInteger("max-monitor-cnt", 300000);
```
Comment (baomingyu) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java:137]: move  "max-monitor-cnt" to  ConfigConstants class

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java [hunk 2]
```diff
@@ -317,19 +363,20 @@ public void run() {
                     if (StringUtils.isBlank(topic)) {
                         blankTopicDiscardMsgCnt.incrementAndGet();
                         takenMsgCnt.decrementAndGet();
+                        monitorIndexExt.incrementAndGet("TUBE_SINK_DROPPED");
```
Comment (baomingyu) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java:366]: "TUBE_SINK_DROPPED"  as a static field

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java [hunk 3]
```diff
@@ -338,8 +385,9 @@ public void run() {
                         }
                     }
                     if (LOG_SINK_TASK_PRINTER.shouldPrint()) {
-                        logger.error("Sink task fail to send the message, finished =" + sendFinished
-                                + ",sink.name=" + Thread.currentThread().getName()
+                        logger.error("Sink task fail to send the message, finished ="
+                                + bChangedInflightValue + ",sink.name="
+                                + Thread.currentThread().getName()
```
Comment (baomingyu) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java:390]: suggest use {} as a placeholder

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java (hunk 1 above)
Comment (gosonzhang) [reply inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java]: done

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java (hunk 2 above)
Comment (gosonzhang) [reply inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java]: done

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java (hunk 3 above)
Comment (gosonzhang) [reply inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java]: done

**Review by baomingyu — APPROVED** [review]
+1

**Review by healchow — APPROVED** [review]

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java [hunk 4]
```diff
@@ -121,6 +137,10 @@ public void configure(Context context) {
         // create producer holder
         producerHolder = new TubeProducerHolder(getName(),
                 usedMasterAddr, configManager.getMqClusterConfig());
+        // get statistic configure items
+        maxMonitorCnt = context.getInteger(MAX_MONITOR_CNT, 300000);
+        statIntervalSec = context.getInteger(STAT_INTERVAL_SEC, 60);
```
Comment (dockerzhang) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java:142]: make the configurations shown in config files.

Comment (gosonzhang) [reply inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java]: done

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**